src/components/new-request/search.vue

```diff
@@ -97,133 +97,141 @@
       </v-col>
 
       <!-- Business Lookup -->
-      <v-col v-if="showBusinessLookup" cols="12">
-        <BusinessLookup />
-      </v-col>
-    </v-row>
```

Comment:
Basic layout cleanup below. Basically, all input fields are now columns inside a single row, and they wrap/stack as needed.